doc/strategy.md

```diff
@@ -0,0 +1,22 @@
+# Goals and strategy
+
+Our goal is to follow these strategic goals and priorities **through March of 2026**, with the plan of re-visiting this structure once we’ve gotten more experience and grown as a team.
+
+## Our 12 month goals
+
+The Foundation began operations this year, and so we define the following 12-month goals:
+
+1. Define the minimal structures needed to efficiently operate  
+2. Make quick progress in using Foundation funds to support Jupyter’s community  
+3. Learn as quickly as we can.  
+4. Prepare ourselves for more complex operations and decision making in the future.
+
+## Our strategy for achieving these goals
+
+Our current strategy is a **short-term strategy**. It is focused on helping us take our first steps, to have initial impact with our funds, and to learn. Here are major strategic priorities:
+
+1. **Focus on learning and taking quick, safe-to-try actions, rather than getting it perfect.** We’re a young team with a lot to learn, and our first job is to learn quickly how we can be most-effective in accomplishing our goals. It’s OK to make mistakes, as long as we learn, and as long as we are doing our best to live up to our goals.  
+2. **Prioritize using funds to deliver value to the Jupyter community.** Prioritize disbursing funds in a way that brings more resources, skills, etc into the Jupyter community. If the Jupyter Foundation needs staff for its own operations, aim to prioritize these for year two so that we can focus on learning for now.  
```

Comment:
```suggestion
2. **Prioritize using funds to deliver value to Project Jupyter.** Prioritize disbursing funds in a way that brings more resources, skills, etc into the Jupyter community. If the Jupyter Foundation needs staff for its own operations, aim to prioritize these for year two so that we can focus on learning for now.  
```